x/twap/api_test.go

```diff
@@ -76,26 +81,43 @@ func (s *TestSuite) TestGetArithmeticTwap() {
 		return getTwapInput{1, quoteAssetDenom, baseAssetDenom, startTime, endTime}
 	}
 
+	makeSimpleTwapToNowInput := func(startTime time.Time, isQuoteTokenA bool) getTwapInput {
```

Comment:
What is the difference from `makeSimpleTwapInput`? Seems to be a 1:1 copy

Reply:
makeSimpleTwapInput takes in the start time, as `GetArithmeticTwap` accepts start time and end time. Meanwhile, `GetArithmeticTwapToNow` does not accept end time,thus only has start time as its parameter